using-mobile-extensions/adobe-target/README.md

```diff
@@ -47,16 +47,54 @@ To add Target to your app:
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
-1. Add the Mobile Core and Target CocoaPods to your project via your `Podfile`.
+{% tab title="iOS (AEP 3.x)" %}
+1. Add the AEPCore, AEPIdentity and AEPTarget CocoaPods to your project via your `Podfile`.
 
    ```text
-    pod 'ACPCore'
-    pod 'ACPTarget'
+    pod 'AEPCore','~>3.0.0'
+    pod 'AEPIdentity','~>3.0.0'
+    pod 'AEPTarget','~>3.0.0'
```

Comment:
This should be ~> 3.0 which implies version up to but not including 4.0.
~3.0.0 would mean up to version 3.1 (not including version 3.1)
Let's fix it.